### PR TITLE
LPD-21633

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-21633

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceReturnContentDisplayContext.java [hunk 1]
```diff
@@ -283,32 +283,68 @@ public CreationMenu getCommerceReturnItemCreationMenu() {
...
-					LiferayWindowState.POP_UP
-				).buildString(),
-				null, "edit", _language.get(httpServletRequest, "edit"), "get",
-				"get", "sidePanel"),
-			new FDSActionDropdownItem(
-				null, null, "delete",
-				_language.get(httpServletRequest, "delete"), "delete", "delete",
-				"headless"));
+		if (StringUtil.equals(commerceReturn.getReturnStatus(), "processing") ||
+			StringUtil.equals(commerceReturn.getReturnStatus(), "completed")) {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceReturnContentDisplayContext.java:289]: Can you use constants for the return status?

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceReturnContentDisplayContext.java [hunk 2]
```diff
@@ -283,32 +283,68 @@ public CreationMenu getCommerceReturnItemCreationMenu() {
...
+					).setParameter(
+						"commerceOrderId", commerceReturn.getOrderId()
+					).setWindowState(
+						LiferayWindowState.POP_UP
+					).buildString(),
+					null, "edit",
+					_language.get(httpServletRequest, "view-refunds"), "get",
+					"get", "modal"));
+		}
+		else if (StringUtil.equals(commerceReturn.getReturnStatus(), "draft")) {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceReturnContentDisplayContext.java:325]: Same here

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra Overall it looks good to me, just a small change. Thank you!

**Comment by andrea-ale-sbarra** [comment]
resent here https://github.com/liferay-commerce/liferay-portal/pull/5100

**Pull request closed without merging**